<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_O"/>
   <addaction name="action_S"/>
   <addaction name="separator"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
   <addaction name="action_7"/>
   <addaction name="action_8"/>
   <addaction name="action_10"/>
   <addaction name="separator"/>
   <addaction name="action_11"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Tools for painting</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose Graph:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="shapeComboBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ellipse</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pen Style:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="penStyleComboBox">
       <item>
        <property name="text">
         <string>Solid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dot</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pen Width:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="penWidthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pen Color:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QComboBox" name="penColorComboBox"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Color for Fill:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QComboBox" name="brushColorComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_N">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/saveas.png</normaloff>:/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="statusTip">
    <string>Save as</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="statusTip">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom-in.png</normaloff>:/images/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="statusTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom-out.png</normaloff>:/images/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="statusTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom-1.png</normaloff>:/images/zoom-1.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="statusTip">
    <string>restore</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Home</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/shear.png</normaloff>:/images/shear.png</iconset>
   </property>
   <property name="text">
    <string>Sheer</string>
   </property>
   <property name="toolTip">
    <string>Sheer</string>
   </property>
   <property name="statusTip">
    <string>Sheer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="statusTip">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/tool.png</normaloff>:/images/tool.png</iconset>
   </property>
   <property name="text">
    <string>Paint Toolbar</string>
   </property>
   <property name="statusTip">
    <string>toolbar</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/help.jpg</normaloff>:/images/help.jpg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>about</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rotate.png</normaloff>:/images/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="statusTip">
    <string>Rotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
